k8s/upgrade/src/plugin/upgrade.rs

```diff
@@ -563,12 +576,89 @@ impl UpgradeResources {
         Ok(())
     }
 
+    /// Create/Delete upgrade config map
+    pub async fn config_map_actions(
```

Comment:
Something like this?

```suggestion
    /// Create/Delete the upgrade config map used to store files specified by the set_file argument.
    pub async fn config_map_actions(
```